playbooks/common/openshift-cluster/upgrades/upgrade_nodes.yml

```diff
@@ -43,83 +35,18 @@
     command: >
       {{ hostvars[groups.oo_first_master.0].openshift.common.admin_binary }} drain {{ openshift.node.nodename | lower }} --force --delete-local-data
     delegate_to: "{{ groups.oo_first_master.0 }}"
```

Comment:
How does everyone feel about keeping drain / mark schedulable out of the role? This implementation seems best to me as it's probably better, maybe even necessary, to control the ordering when we get to including it in control plane upgrade.